### PR TITLE
NavCoin.org Russian and Chinese Translations

### PR DESCRIPTION
**Pull request by matt-auckland**
Visit these links to view the translations

https://deploy-preview-137--navcoin.netlify.com/ru/

https://deploy-preview-137--navcoin.netlify.com/zh/

### CODE REVIEW

File: content/get-involved/index.zh.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: "为协议做贡献"
+date: 2018-03-20T07:41:22+13:00
+draft:false
```
Comment (matt-auckland) [inline content/get-involved/index.zh.md]:
```suggestion
draft: false
```

Comment (IgorsLV) [reply content/get-involved/index.zh.md]: Space is added

File: content/merchants/index.zh.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        imgSrc="/images/merchants/mer-1gbits.png"
+        text="在超过15个不同的国家拥有20多个世界级的服务器&nbsp;中心。"
+        linkUrl="https://1gbits.com/"
+    >}}
+{{< /item_container_section >}}
+
+{{< item_container_section
+    titleText="接受NavCoin很容易!"
+    subtitleText="NavCoin为您的企业提供了正确的支付解决&nbsp;方案。"
+    bgPurple=""true
```
Comment (matt-auckland) [inline content/merchants/index.zh.md]:
```suggestion
    bgPurple="true"
```

File: content/governance/index.zh.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+{{< /zig_section>}}
+
+
+{{< zig_section
+  titleText="让世界变得更好"
+  imgSrc="/images/governance/gov-world.svg"
+  bgPurple="true"
+>}}
+像NavCoin这样的去中心化数字货币从政府和企业精英手中夺走了对货币体系的控制权，将自由、隐私和金融权力还给了个人。这就是为什么NavCoin不仅仅是一个支付协议，它实际上是全球经济系统运作方式发生深刻变化的一个载体。去中心啊乎的治理是其核心。由于NavCoin没有中央权威，作为这个社区的成员，我们的一个关键角色是倡导中立和基于共识的&nbsp;决策。
+{{< /zig_section
```
Comment (matt-auckland) [inline content/governance/index.zh.md]:
```suggestion
{{< /zig_section }}
```

File: content/governance/index.zh.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+{{< /zig_section>}}
+
+
+{{< zig_section
+  titleText="让世界变得更好"
+  imgSrc="/images/governance/gov-world.svg"
+  bgPurple="true"
+>}}
+像NavCoin这样的去中心化数字货币从政府和企业精英手中夺走了对货币体系的控制权，将自由、隐私和金融权力还给了个人。这就是为什么NavCoin不仅仅是一个支付协议，它实际上是全球经济系统运作方式发生深刻变化的一个载体。去中心啊乎的治理是其核心。由于NavCoin没有中央权威，作为这个社区的成员，我们的一个关键角色是倡导中立和基于共识的&nbsp;决策。
+{{< /zig_section }}
```
Comment (matt-auckland) [inline content/governance/index.zh.md]: I missed out a `>` here, my bad
```suggestion
{{< /zig_section >}}
```

**Comment by matt-auckland** [comment]
I just noticed that the homepage isn't actually translated in either language 😰
My script must not duplicate those files.
I will make a pull request that adds them.

**Comment by IgorsLV** [comment]
Ok

вс, 13 янв. 2019 г. в 22:34, Matt <notifications@github.com>:

File: content/_index.zh.html [hunk 5]
```diff
@@ -0,0 +1,224 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+<div class="homepage">
+  <section class="hero">
+    <div class="grid-container">
+      <img src="/images/homepage/home hero blue_v2@2x.png" alt="Hero image">
+      <div class="text-container">
+        <h1>简化加密</h1>
+        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做&nbsp;生意的&nbsp;成本</p>
+        <a href="/en/wallets/" class="round-btn gradient-btn">Wallets <img src="/images/icons/rightward-arrow.svg" alt="rightward arrow" class="button-icon"></a>
```
Comment (matt-auckland) [inline content/_index.zh.html]: This section needs translation

File: content/_index.zh.html [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+  </div>
+  <section class="welcome" id="welcome">
+    <h1>欢迎来到NavCoin</h1>
+    <div class="grid-container ninety-vw-container">
+      <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img">
+      <ul class="text-container">
+        <li class="">
+          <img src="/images/homepage/hom-fast-cheap.svg" alt="">
+          <h2>低摩擦支付</h2>
+          <p>专为快速、实惠和安全的对等支付而设计。一种旨在降低经营成本&nbsp;的&nbsp;网络。</p>
```
Comment (matt-auckland) [inline content/_index.zh.html]: This section sounds like it has been google translated

File: content/_index.zh.html [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-idea.svg" alt="">
+          <h2>2014年开始社区运营</h2>
+          <p>去中心化不只是一个流行词。它建立在网络的基础之上，创造一种未经许可的&nbsp;i创新&nbsp;文化。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-cfund.svg" alt="">
+          <h2>一种为自己买单的经济&nbsp;体系。</h2>
+          <p>NavCoin本身使用的是有史以来最先进的社区基金。第一个完全去中心化、完全自治、只有社区才能&nbsp;控制的&nbsp;基金。</p>
```
Comment (matt-auckland) [inline content/_index.zh.html]: This section sounds like it has been google translated

File: content/_index.zh.html [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+            </div>
+            <div class="div-5">
+              <img src="/images/homepage/hom-fast.svg" alt="">
+              <h2>快速支付</h2>
+              <p>交易在几秒钟内进行，30秒内得到确认，这对业务来说&nbsp;是&nbsp;理想的。</p>
+            </div>
+            <div class="div-6">
+              <img src="/images/homepage/hom-oa.svg" alt="">
+              <h2>支付一个名字，而不是一个数字</h2>
+              <p>NavCoin使用OpenAlias，发送钱就像输入电子邮件地址&nbsp;一样&nbsp;简单。</p>
```
Comment (matt-auckland) [inline content/_index.zh.html]: This section sounds like it has been google translated or needs improving

File: content/_index.zh.html [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+    <h1>建立在网络中的价值。</h1>
+      <ul class="grid-container text-container ninety-vw-container">
+        <li class="">
+          <img src="/images/homepage/hom-governance.svg" alt="">
+          <h2>公开透明的治理</h2>
+          <p>没有人能控制NavCoin——每个人都可以自由贡献他们想要的东西。没有强迫的改变，每个钱包在NavCoin的方向上都&nbsp;有&nbsp;发言权。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-open-source.svg" alt="">
+          <h2一切开源</h2>
```
Comment (matt-auckland) [inline content/_index.zh.html]:
```suggestion
          <h2>一切开源</h2>
```

File: content/for-individuals/index.zh.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+  >}}
+  {{< step
+      titleText="第四部：安全的网络"
+      imgSrc="/images/for-beginners/beg-secure-network.svg"
+      text="如果您正在运行NavCoin核心钱包，那么您可以选择加注您的硬币，并获得每年4%的加注奖励。有一系列的设备可以使标记变得更容易，并且社区创建了许多信息来帮助您&nbsp;设置。"
+  >}}
+  {{< step
+      titleText="第5步：花费 NavCoin"
+      imgSrc="/images/for-beginners/beg-spend.svg"
+      text="你可以用你的NavCoin在网上和商店里买各种各样的东西。看看这些接受导航和显示你的支持的&nbsp;商家。"
```
Comment (matt-auckland) [inline content/for-individuals/index.zh.md]: This section sounds like it has been google translated or needs improving

File: content/for-developers/index.zh.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+        buttonText="Learn more"
+        buttonImgSrc="/images/icons/rightward-arrow.svg"
+    >}}
+{{< /item_container_section >}}
+
+{{< contrib_list
+    titleText="NavCoin协议贡献者"
+    subtitleText="我们要感谢我们的贡献者，他们让这个协议变得非常&nbsp;棒!"
+    buttonUrl="https://github.com/NAVCoin/navcoin-core/graphs/contributors"
+    buttonTxt="View NavCoin on GitHub"
```
Comment (matt-auckland) [inline content/for-developers/index.zh.md]: Needs translation

File: content/for-business/index.zh.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+        imgSrc="/images/for-business/bus-low-fee.svg"
+        text="NavCoin是世界上最便宜的支付网络之一，收费通常低于&nbsp;1%。"
+    >}}
+    {{< feature                 
+        titleText="环保"
+        imgSrc="/images/for-business/bus-eco.svg"
+        text="只消耗其他网络一小部分能量的&nbsp;支付网络。"
+    >}}
+    {{< feature                 
+        titleText="Flexibility"
```
Comment (matt-auckland) [inline content/for-business/index.zh.md]: Needs translation

File: content/for-business/index.zh.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+>}}
+集成导航按钮进行快速支付是一个很容易实现的选项，适合个人或副业。这就像复制粘贴一些代码到你的网站上一样简单。你可以选择嵌入按钮，或者二维码，这样更容易&nbsp;分享。
+{{< /zig_section>}}
+
+{{< center_text_cta
+    lightStyle="正确的"
+    titleText="哪些人在接受 NAV?"
+    buttonText="可以在这里参看商人"
+    buttonImgSrc="/images/icons/rightward-arrow.svg"
+    buttonLink="/商人/"
```
Comment (matt-auckland) [inline content/for-business/index.zh.md]: This link text should not be modified
```suggestion
    buttonLink="/merchants/"
```

File: content/for-business/index.zh.md [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+  buttonUrl="https://brianium.github.io/nav-button/"
+  buttonImgSrc="/images/icons/rightward-arrow.svg"
+  bgPurple="true"
+  newTab="true"
+>}}
+集成导航按钮进行快速支付是一个很容易实现的选项，适合个人或副业。这就像复制粘贴一些代码到你的网站上一样简单。你可以选择嵌入按钮，或者二维码，这样更容易&nbsp;分享。
+{{< /zig_section>}}
+
+{{< center_text_cta
+    lightStyle="正确的"
```
Comment (matt-auckland) [inline content/for-business/index.zh.md]:
```suggestion
    lightStyle="true"
```
this should not be translated

File: content/buy-navcoin/index.zh.md [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+date: 2018-03-20T08:09:08+13:00
+draft: false
+type: "default"
+author: ""
+description: "是公开开采的，没有代币销售。购买NavCoin的唯一方法是从第三方或交易所购买。"
+---
+{{< hero_section
+titleText="购买NavCoin"
+imgSrc="/images/buy-navcoin/buy-hero.svg"
+paragraphText="NavCoin是公开开采的，没有代币销售。获得资产净值的唯一方法是从第三方或交易所购买。交换提供了高度不同程度的安全性、安全性、隐私以及对您的资金和信息的控制。执行您自己的尽职调查，并选择一个钱包，您将保留您的NavCoin在选择一个交易所。请按此连结查阅兑换金额，以获取更多资讯。"
```
Comment (matt-auckland) [inline content/buy-navcoin/index.zh.md]: This section sounds like it has been google translated or needs improving

File: content/get-involved/index.zh.md [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+reversed="true"
+newTab="true"
+>}}
+对于NavCoin Core的所有非关键问题，请搜索类似的问题，如果没有发现，请提交一个新问题，提供以下&nbsp;信息。
+<br>
+<ul class="article-ul" style="color: rgba(0, 0, 0, 0.55);">
+  <li>对问题的清晰描述以及重现问题的&nbsp;步骤。</li>
+  <li>您使用的是什么版本的NavCoin Core，或者您使用什么提交构建的.</li><li>从您的debug.log 文件中来的任何相关条目。请确保在发布您的debug.log之前已经删除任何敏感 &nbsp;信息。</li>
+</ul>
+<p class="paragraph-text">The best strategy to get your issue fixed quickly is to make it as easy as possible for the development team to 跟踪问题并编写修复程序。提供更多的信息并组织好这些信息将大有&nbsp;帮助。</p>
```
Comment (matt-auckland) [inline content/get-involved/index.zh.md]: Needs translation

File: content/wallets/index.zh.md [hunk 17]
```diff
@@ -0,0 +1,155 @@
...
+    subtitleText="NavCoin Core钱包经过精心提炼，直观易用。它附带了一个安装向导，以及加载区块链并开始在安全环境中发送交易所需的&nbsp;一切。"
+    bgPurple="正确的"
+    anchor="download-core"
+    buttonUrl="https://github.com/NAVCoin/navcoin-core/releases/latest"
+    buttonText="先进的下载"
+    buttonImgSrc="/images/icons/rightward-arrow.svg"
+    newTab="正确的"
+>}}
+    {{< action
+        titleText="窗口"
```
Comment (matt-auckland) [inline content/wallets/index.zh.md]: This is Windows the operating system, not an actual window. Translation is incorrect

File: content/wallets/index.zh.md [hunk 18]
```diff
@@ -0,0 +1,155 @@
...
+    >}}
+{{< /item_container_section >}}
+
+{{< item_container_section
+    titleText="下载NavCoin Core 4.5.0"
+    subtitleText="NavCoin Core钱包经过精心提炼，直观易用。它附带了一个安装向导，以及加载区块链并开始在安全环境中发送交易所需的&nbsp;一切。"
+    bgPurple="正确的"
+    anchor="download-core"
+    buttonUrl="https://github.com/NAVCoin/navcoin-core/releases/latest"
+    buttonText="先进的下载"
```
Comment (matt-auckland) [inline content/wallets/index.zh.md]: This section sounds like it has been google translated or needs improving

File: content/_index.zh.html (hunk 9 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/for-business/index.zh.md (hunk 13 above)
Comment (IgorsLV) [reply content/for-business/index.zh.md]: fixed

File: content/_index.zh.html (hunk 5 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/for-developers/index.zh.md (hunk 11 above)
Comment (IgorsLV) [reply content/for-developers/index.zh.md]: fixed

File: content/for-business/index.zh.md (hunk 12 above)
Comment (IgorsLV) [reply content/for-business/index.zh.md]: fixed

File: content/wallets/index.zh.md (hunk 17 above)
Comment (IgorsLV) [reply content/wallets/index.zh.md]: fixed

File: content/wallets/index.zh.md (hunk 18 above)
Comment (IgorsLV) [reply content/wallets/index.zh.md]: fixed

File: content/_index.zh.html (hunk 6 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 7 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 8 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/for-individuals/index.zh.md (hunk 10 above)
Comment (IgorsLV) [reply content/for-individuals/index.zh.md]: fixed

File: content/buy-navcoin/index.zh.md (hunk 15 above)
Comment (IgorsLV) [reply content/buy-navcoin/index.zh.md]: fixed

File: content/get-involved/index.zh.md (hunk 16 above)
Comment (IgorsLV) [reply content/get-involved/index.zh.md]: fixed

File: content/_index.zh.html [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+        <h2>1. 协议</h2>
+        <hr>
+        <p>NavCoin区块链是一切的基础，它是完全开源的，由运行在世界各地的节点&nbsp;来&nbsp;保护。</p>
+        <a href="/en/get-involved/" class="round-btn white-btn-new">参与进来<img src="/images/icons/rightward-arrow-purple.svg" alt="rightward arrow" class="button-icon"></a>
+      </div>
+      <div class="column">
+        <img src="/images/homepage/hom-navhub.svg" alt="collection of blocks" class="large-img">
+        <h2>2. NavHub</h2>
+        <hr>
+        <p>NavHub是我们生态系统的第二层，在这里你可以看到各种各样的项目正在navcoin协议的基础&nbsp;上&nbsp;构建。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
        <p>NavHub是我们生态系统的第二层，在这里你可以看到各种各样的项目正在navcoin协议的基础上&nbsp;构建。</p>
```

File: content/_index.zh.html [hunk 20]
```diff
@@ -0,0 +1,224 @@
...
+        <p>任何人都可以参与NavCoin，甚至可以获得报酬。加入我们的社区频道，收听最新的新闻&nbsp;和&nbsp;更新。</p>
+        <a href="#community-id" class="round-btn white-btn-new">Join Community<img src="/images/icons/downward-arrow-purple.svg" alt="downward arrow" class="button-icon"></a>
+      </div>
+    </div>
+
+  </section>
+  <section id="community-id" class="community">
+    <div class="spacer four"></div>
+    <h1 style="margin-top: 16px;">加入我们积极的NavCoin&nbsp;社区 </h1>
+    <p>任何人都可以加入我们的实干家社区:开发人员、企业家和技术专家共同努力建立一个新的&nbsp;经济&nbsp;体系。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
    <p>任何人都可以加入我们的实干家社区:开发人员、企业家和技术专家共同努力建立一个新的经济&nbsp;体系。</p>
```

File: content/_index.zh.html [hunk 21]
```diff
@@ -0,0 +1,224 @@
...
+        <h2>2. NavHub</h2>
+        <hr>
+        <p>NavHub是我们生态系统的第二层，在这里你可以看到各种各样的项目正在navcoin协议的基础&nbsp;上&nbsp;构建。</p>
+        <a href="https://navhub.org/" class="round-btn white-btn-new" target="_blank">浏览 NavHub <img src="/images/icons/rightward-arrow-purple.svg" alt="rightward arrow" class="button-icon"></a>
+      </div>
+      <div class="column">
+        <img src="/images/homepage/hom-people.svg" alt="connected network of blocks" class="large-img">
+        <h2>3. 人们</h2>
+        <hr>
+        <p>任何人都可以参与NavCoin，甚至可以获得报酬。加入我们的社区频道，收听最新的新闻&nbsp;和&nbsp;更新。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
        <p>任何人都可以参与NavCoin，甚至可以获得报酬。加入我们的社区频道，收听最新的新闻和&nbsp;更新。</p>
```

File: content/_index.zh.html [hunk 22]
```diff
@@ -0,0 +1,224 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+<div class="homepage">
+  <section class="hero">
+    <div class="grid-container">
+      <img src="/images/homepage/home hero blue_v2@2x.png" alt="Hero image">
+      <div class="text-container">
+        <h1>简化加密</h1>
+        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做&nbsp;生意的&nbsp;成本</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做生意的&nbsp;成本</p>
```

File: content/_index.zh.html [hunk 23]
```diff
@@ -0,0 +1,224 @@
...
+      <ul class="text-container">
+        <li class="">
+          <img src="/images/homepage/hom-fast-cheap.svg" alt="">
+          <h2>低摩擦支付</h2>
+          <p>为实现快捷、实惠和安全的对等支付而进行专门设计的网络。其宗旨在于降低经营网络&nbsp的&nbsp;成本。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-idea.svg" alt="">
+          <h2>2014年开始社区运营</h2>
+          <p>去中心化不只是一个流行词。它建立在网络的基础之上，创造一种未经许可的&nbsp;i创新&nbsp;文化。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
          <p>去中心化不只是一个流行词。它建立在网络的基础之上，创造一种未经许可的创新&nbsp;文化。</p>
```

File: content/_index.zh.html [hunk 24]
```diff
@@ -0,0 +1,224 @@
...
+  </div>
+  <section class="welcome" id="welcome">
+    <h1>欢迎来到NavCoin</h1>
+    <div class="grid-container ninety-vw-container">
+      <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img">
+      <ul class="text-container">
+        <li class="">
+          <img src="/images/homepage/hom-fast-cheap.svg" alt="">
+          <h2>低摩擦支付</h2>
+          <p>为实现快捷、实惠和安全的对等支付而进行专门设计的网络。其宗旨在于降低经营网络&nbsp的&nbsp;成本。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
          <p>为实现快捷、实惠和安全的对等支付而进行专门设计的网络。其宗旨在于降低经营网络的&nbsp;成本。</p>
```

File: content/_index.zh.html [hunk 25]
```diff
@@ -0,0 +1,224 @@
...
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-cfund.svg" alt="">
+          <h2>一种为自己买单的经济&nbsp;体系。</h2>
+          <p>NavCoin自身使用有史以来最先进的社区基金，它是第一个完全去中心化、完全自治、实现社区掌权的&nbsp;基金。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-friendly.svg" alt="">
+          <h2>人性化的钱</h2>
+          <p>加密货币很复杂，但它不需要如此。NavCoin的目标是让你的父母用密码支付变得&nbsp;非常&nbsp;容易。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
          <p>加密货币很复杂，但它不需要如此。NavCoin的目标是让你的父母用密码支付变得非常&nbsp;容易。</p>
```

File: content/_index.zh.html [hunk 26]
```diff
@@ -0,0 +1,224 @@
...
+          <div class="grid-container ninety-vw-container">
+            <div class="div-1">
+              <img src="/images/homepage/hom-low-energy.svg" alt="">
+              <h2>可持续股权证明</h2>
+              <p>NavCoin使用的是股权证明，消耗的能量只相当于工作网络证明的&nbsp;一&nbsp;小部分。</p>
+            </div>
+            <div class="div-2">
+              <img src="/images/homepage/hom-scalable.svg" alt="">
+              <h2>网络可伸缩性</h2>
+              <p>测试显示当前的处理能力为每秒260多个交易，同时积极研究第二层的扩展&nbsp;解决&nbsp;方案。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
              <p>测试显示当前的处理能力为每秒260多个交易，同时积极研究第二层的扩展解决&nbsp;方案。</p>
```

File: content/_index.zh.html [hunk 27]
```diff
@@ -0,0 +1,224 @@
...
+      <div></div>
+    </div>
+    <div class="padding">
+      <h1>NavCoin的高级功能</h1>
+      <div class="center">
+          <div class="grid-container ninety-vw-container">
+            <div class="div-1">
+              <img src="/images/homepage/hom-low-energy.svg" alt="">
+              <h2>可持续股权证明</h2>
+              <p>NavCoin使用的是股权证明，消耗的能量只相当于工作网络证明的&nbsp;一&nbsp;小部分。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
              <p>NavCoin使用的是股权证明，消耗的能量只相当于工作网络证明的一&nbsp;小部分。</p>
```

File: content/_index.zh.html [hunk 28]
```diff
@@ -0,0 +1,224 @@
...
+            </div>
+            <div class="div-4">
+              <img src="/images/homepage/hom-privacy.svg" alt="">
+              <h2>隐私</h2>
+              <p>从最初的NavTech到后来的升级，隐私一直是NavCoin的&nbsp;核心&nbsp;部分。</p>
+            </div>
+            <div class="div-5">
+              <img src="/images/homepage/hom-fast.svg" alt="">
+              <h2>快速支付</h2>
+              <p>交易在几秒钟内进行，30秒内得到确认，这对业务来说&nbsp;是&nbsp;理想的。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
              <p>交易在几秒钟内进行，30秒内得到确认，这对业务来说是&nbsp;理想的。</p>
```

File: content/_index.zh.html [hunk 29]
```diff
@@ -0,0 +1,224 @@
...
+            </div>
+            <div class="div-3">
+              <img src="/images/homepage/hom-international.svg" alt="">
+              <h2>低成本全球化支付</h2>
+              <p>把钱发送到世界上任何地方都很便宜，不需要&nbsp;经过&nbsp;银行。</p>
+            </div>
+            <div class="div-4">
+              <img src="/images/homepage/hom-privacy.svg" alt="">
+              <h2>隐私</h2>
+              <p>从最初的NavTech到后来的升级，隐私一直是NavCoin的&nbsp;核心&nbsp;部分。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
              <p>从最初的NavTech到后来的升级，隐私一直是NavCoin的核心&nbsp;部分。</p>
```

File: content/_index.zh.html [hunk 30]
```diff
@@ -0,0 +1,224 @@
...
+            </div>
+            <div class="div-2">
+              <img src="/images/homepage/hom-scalable.svg" alt="">
+              <h2>网络可伸缩性</h2>
+              <p>测试显示当前的处理能力为每秒260多个交易，同时积极研究第二层的扩展&nbsp;解决&nbsp;方案。</p>
+            </div>
+            <div class="div-3">
+              <img src="/images/homepage/hom-international.svg" alt="">
+              <h2>低成本全球化支付</h2>
+              <p>把钱发送到世界上任何地方都很便宜，不需要&nbsp;经过&nbsp;银行。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
              <p>把钱发送到世界上任何地方都很便宜，不需要经过&nbsp;银行。</p>
```

File: content/_index.zh.html [hunk 31]
```diff
@@ -0,0 +1,224 @@
...
+            </div>
+            <div class="div-5">
+              <img src="/images/homepage/hom-fast.svg" alt="">
+              <h2>快速支付</h2>
+              <p>交易在几秒钟内进行，30秒内得到确认，这对业务来说&nbsp;是&nbsp;理想的。</p>
+            </div>
+            <div class="div-6">
+              <img src="/images/homepage/hom-oa.svg" alt="">
+              <h2>支付一个名字，而不是一个数字</h2>
+              <p>NavCoin应用OpenAlias，让发送钱就如输入电子邮件地址&nbsp;一样的&nbsp;简单。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
              <p>NavCoin应用OpenAlias，让发送钱就如输入电子邮件地址一样的&nbsp;简单。</p>
```

File: content/_index.zh.html [hunk 32]
```diff
@@ -0,0 +1,224 @@
...
+      <ul class="grid-container text-container ninety-vw-container">
+        <li class="">
+          <img src="/images/homepage/hom-governance.svg" alt="">
+          <h2>公开透明的治理</h2>
+          <p>没有人能控制NavCoin——每个人都可以自由贡献他们想要的东西。没有强迫的改变，每个钱包在NavCoin的方向上都&nbsp;有&nbsp;发言权。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-open-source.svg" alt="">
+          <h2>一切开源</h2>
+          <p>开源是NavCoin的支柱。每一行代码都是可见的，任何人都可以为网站做贡献。信任必须通过透明&nbsp;来&nbsp;建立。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
          <p>开源是NavCoin的支柱。每一行代码都是可见的，任何人都可以为网站做贡献。信任必须通过透明来&nbsp;建立。</p>
```

File: content/_index.zh.html [hunk 33]
```diff
@@ -0,0 +1,224 @@
...
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-vote.svg" alt="">
+          <h2>成为你自己的银行</h2>
+          <p>作为一个对等网络意味着没有银行或中介机构的阻碍。一种你的选票很重要的&nbsp;经济&nbsp;制度。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
          <p>作为一个对等网络意味着没有银行或中介机构的阻碍。一种你的选票很重要的经济&nbsp;制度。</p>
```

File: content/_index.zh.html [hunk 34]
```diff
@@ -0,0 +1,224 @@
...
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-new-system.svg" alt="">
+          <h2>建立新制度</h2>
+          <p>NavCoin是整个全球经济体系运作方式变化的象征——一个将自由、隐私和金融权力归还给个人&nbsp;的&nbsp;体系。</p>
```
Comment (kakax114) [inline content/_index.zh.html]:
```suggestion
          <p>NavCoin是整个全球经济体系运作方式变化的象征——一个将自由、隐私和金融权力归还给个人的&nbsp;体系。</p>
```

Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 33 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 32 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 31 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 30 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 29 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 28 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 27 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 26 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 25 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 24 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 23 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 22 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 21 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 20 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html (hunk 19 above)
Comment (IgorsLV) [reply content/_index.zh.html]: fixed

File: content/_index.zh.html [hunk 35]
```diff
@@ -0,0 +1,224 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+<div class="homepage">
+  <section class="hero">
+    <div class="grid-container">
+      <img src="/images/homepage/home hero blue_v2@2x.png" alt="Hero image">
+      <div class="text-container">
+        <h1>简化加密</h1>
+        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做生意的&nbsp;成本</p>
```
Comment (marcus290) [inline content/_index.zh.html]:
```suggestion
        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做生意的&nbsp;成本。</p>
```

Comment (IgorsLV) [reply content/_index.zh.html]: done and &nbsp; code in Chinese files is deleted

**Pull request merged**